### PR TITLE
Comments suggestion

### PR DESCRIPTION
**Pull request by Dcruise546**
Have a look at my suggestions! Do the changes and assign me again as reviewer!

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -203,24 +215,23 @@ def collision():
 	# Event loop
 	for event in pygame.event.get():
 		
-		if event.type == pygame.QUIT or event.type == pygame.KEYDOWN:
+		if event.type == pygame.QUIT or event.type == pygame.KEYDOWN:		# bug to be fixed
```
Comment (Dcruise546) [inline main.py]: If you know the existence of a bug, then either fix it or keep this outside 'main' branch. For eg, keep it on your branch and merge it only after fixing this!
I will approve this just for one time!

File: main.py [hunk 2]
```diff
@@ -184,17 +196,17 @@ def collision():
 # GAME LOOP
 while run:
 	# Draws the stuff to be displayed in window
-	bgX, bg_width, groundX, ground_width = redrawWindow(bgX, bg_width, groundX, ground_width)
+	bgX, bg_width, groundX, ground_width = draw_stuff_on_window(bgX, bg_width, groundX, ground_width)
```
Comment (Dcruise546) [inline main.py]: Draw what 'stuff'? It should be more clear. Choose another word.

File: main.py [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
 import random
+import sys
 import time
-import json
 
-class Player(object):
-	
-	imgs = [pygame.image.load(os.path.join('pics', "player-"+ str(x) + '.png')) for x in range(1,10)]
+
+# The class Player descibes the the player object. It has info such as the x positon, y position, width, height, contour coordinates and
```
Comment (Dcruise546) [inline main.py]: remove 'the the'

File: main.py [hunk 4]
```diff
@@ -1,15 +1,18 @@
...
+# a draw method to draw the player's images onto the screen hence creating the animation effect.
+class Player:
+	# Loading player images
+	num_of_player_imgs = 9
```
Comment (Dcruise546) [inline main.py:16]: Either use camel case (numOfPlayers) or snake case (num_of_players) and not both. Maintain any one style throughout your program.

File: main.py [hunk 5]
```diff
@@ -184,17 +196,17 @@ def collision():
...
 
 	# Gets the mouse pointer coordinates
 	(mx, my) = pygame.mouse.get_pos()
 
 	# Displaying the mouse pointer coordinates for reference
-	display_pointer(mx,my)
+	display_mouse_pointer_coordinates(mx,my)
 	   	
-	# To limit the movable region
+	# To limit the movable region of player
```
Comment (Dcruise546) [inline main.py]: Comments should be as minimal as possible and still express everything the programmer wants to convey.
For eg: 'limit players movable region'

**Review by Dcruise546 — COMMENTED** [review]
Please work on it and send it back again!

File: main.py (hunk 1 above)
Comment (Sowbhagya-lakshmi) [reply main.py]: Ok, senior. It is fixed now.

**Pull request merged**